CMakeLists.txt

```diff
@@ -26,6 +26,10 @@ cmake_minimum_required(VERSION 3.12)
 
 project(OpenSubdiv)
 
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
This might be best set if this is the root project? (Not consumed as a sub-project?) Or will this be scoped to this project only?

Reply:
Setting `CMAKE_CXX_STANDARD` as a normal variable will only impact the current directory, but it is good practice to let the consumers define it themselves so that they can build everything with a consistent standard. Setting it only when built as a standalone project won't allow people to customize it with `cmake -DCMAKE_CXX_STANDARD=17`, so a preferred solution is to either define it only if not defined, or to set it as a cache variable.

Solution 1:
```if (NOT DEFINED CMAKE_CXX_STANDARD)
  set(CMAKE_CXX_STANDARD 14)
endif()```

 - allows customizing the standard with `cmake -DCMAKE_CXX_STANDARD=17` or with `set(CMAKE_CXX_STANDARD 17)` in a parent project.

Solution 2:
```set(CMAKE_CXX_STANDARD 14 CACHE STRING "C++ standard to build with")```

 - displays an entry in tools such as `ccmake` or `cmake-gui`, which can facilitate project configuration
 - allows customizing with `cmake -DCMAKE_CXX_STANDARD=17` or with `set(CMAKE_CXX_STANDARD 17 CACHE STRING "")` in a parent project (if it uses `cmake < 3.21`) or with
 ```set(CMAKE_POLICY_DEFAULT_CMP0126 NEW) # Avoid overriding normal variables with set(CACHE)
 set(CMAKE_CXX_STANDARD 17)```
 in a parent project (if it uses `cmake >= 3.21, see policy [CMP0126](https://cmake.org/cmake/help/latest/policy/CMP0126.html).